tests/test_keras_to_tensorflow.py

```diff
@@ -63,113 +80,28 @@ def assert_converted_model(tf_model_dir):
-    assert_converted_model(tf_model_dir)
-    restart_serving_container(model_name)
-    assert_model_serving(model_name, imagenet_dictionary, [
-        ('tiger cat', 0.334694504737854),
-        ('Egyptian cat', 0.2851393222808838),
-        ('tabby, tabby cat', 0.15471667051315308),
-        ('kit fox, Vulpes macrotis', 0.03160465136170387),
-        ('lynx, catamount', 0.030886519700288773)
-    ])
-
-
-def test_convert_imagenet_resnet50(temp_file, imagenet_dictionary):
-    model_name = 'resnet50'
-    tf_model_dir = setup_model(model_name, temp_file)
-    KerasToTensorflow.convert(temp_file, tf_model_dir)
-    assert_converted_model(tf_model_dir)
-    restart_serving_container(model_name)
-    assert_model_serving(model_name, imagenet_dictionary, [
-        ('red fox, Vulpes vulpes', 0.3193315863609314),
-        ('kit fox, Vulpes macrotis', 0.19359852373600006),
-        ('weasel', 0.14291106164455414),
-        ('Pembroke, Pembroke Welsh corgi', 0.1395975947380066),
-        ('lynx, catamount', 0.04618712514638901)
-    ])
-
-
-def test_convert_imagenet_xception(temp_file, imagenet_dictionary):
-    model_name = 'xception'
-    tf_model_dir = setup_model(model_name, temp_file)
-    KerasToTensorflow.convert(temp_file, tf_model_dir)
-    assert_converted_model(tf_model_dir)
-    restart_serving_container(model_name)
-    assert_model_serving(model_name, imagenet_dictionary, [
-        ('red fox, Vulpes vulpes', 0.10058529675006866),
-        ('weasel', 0.09152575582265854),
-        ('Pembroke, Pembroke Welsh corgi', 0.07581676542758942),
-        ('tiger cat', 0.0746716633439064),
-        ('kit fox, Vulpes macrotis', 0.06751589477062225)
-    ])
-
-
-def test_convert_imagenet_vgg16(temp_file, imagenet_dictionary):
-    model_name = 'vgg16'
-    tf_model_dir = setup_model(model_name, temp_file)
-    KerasToTensorflow.convert(temp_file, tf_model_dir)
-    assert_converted_model(tf_model_dir)
-    restart_serving_container(model_name)
-    assert_model_serving(model_name, imagenet_dictionary, [
-        ('kit fox, Vulpes macrotis', 0.3090206980705261),
-        ('red fox, Vulpes vulpes', 0.21598483622074127),
-        ('Egyptian cat', 0.1327403038740158),
-        ('tiger cat', 0.11005250364542007),
-        ('tabby, tabby cat', 0.08285804092884064)
-    ])
-
-
-def test_convert_imagenet_vgg19(temp_file, imagenet_dictionary):
-    model_name = 'vgg19'
-    tf_model_dir = setup_model(model_name, temp_file)
-    KerasToTensorflow.convert(temp_file, tf_model_dir)
-    assert_converted_model(tf_model_dir)
-    restart_serving_container(model_name)
-    assert_model_serving(model_name, imagenet_dictionary, [
-        ('red fox, Vulpes vulpes', 0.3812929391860962),
-        ('kit fox, Vulpes macrotis', 0.27262774109840393),
-        ('tiger cat', 0.08553500473499298),
-        ('lynx, catamount', 0.05379556491971016),
-        ('Egyptian cat', 0.047869954258203506)
-    ])
+def test_convert_tests_cover_all_model_types():
```

Comment:
s/conver/convert

Reply:
It's cover, as in all the model types are tested